### DISCUSSION TITLE
Can I remove the script function "Adding to collection" feature?

### DISCUSSION THREAD

**Discussion by CrocodileDuty** [Taxel/PlexTraktSync#1495] [Support] [answered]
It doesnt show much value for me.. I would just like to use the sync feature.
 I often tend to add and remove content from my server and the sync wont remove collections that are already added in trakt right?

**Comment by RileyXX** [comment]
There is a config setting to clear collected items when they are deleted. Its disabled by default

**Reply by CrocodileDuty** [reply]
Is there any way to completely remove this "Adding to collection" feature from the script? Just sync my watching and stuff and forget about adding stuffs to collection? It makes the script very time taking.

**Reply by simonc56** [reply, marked as answer]
Please read README file : https://github.com/Taxel/PlexTraktSync#sync-settings
